examples/advanced/quantum_volume.py

```diff
@@ -0,0 +1,147 @@
+"""Tool to run the Quantum Volume benchmark defined by IBM in
+https://arxiv.org/abs/1811.12926."""
```

Comment:
Include usage instructions and example output.

Reply:
Done, although this will be updated as the program is completed.